ddp-workspace/projects/ddp-sdk/src/lib/components/languageSelector.component.ts

```diff
@@ -0,0 +1,102 @@
+import { Component, Input, OnInit } from "@angular/core";
+import { StudyLanguage } from "../models/studyLanguage";
+import { LanguageServiceAgent } from "../services/serviceAgents/languageServiceAgent.service";
+import { isNullOrUndefined } from "util";
+import { TranslateService } from "@ngx-translate/core";
+
+@Component({
+  selector: 'ddp-language-selector',
+  template: `
+    <div [hidden]="!loaded">
+      <button class="SimpleButton" [ngClass]="{'SimpleButton--Scrolled': isScrolled}" *ngIf="currentLanguage !== null && currentLanguage !== undefined" [matMenuTriggerFor]="menu">
+        <svg class="ddp-globe" height="24px" width="24px" aria-live="title">
+          <title id="title" [lang]="currentLanguage.languageCode" translate>Toolkit.Header.LanguageSelection</title>
+          <use href="assets/images/globe.svg#Language-Selector-3"></use>
```

Comment:
I think we can make icon URL configurable, we can add a new property in the ConfigurationService with the default value. And if someone wants to change the icon URL, he can do it under his project.